### PR TITLE
update faq with common ask questions

### PR DESCRIPTION
**Pull request by sj5n**
*Issue #, if available:*

*Description of changes:*
Update FAQ with commonly asked questions.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/partner_editable/faq_troubleshooting.adoc [hunk 1]
```diff
@@ -11,10 +11,117 @@ WARNING: When you set *Rollback on failure* to *Disabled*, you continue to incur
...
+
+---
+
+*Q.* Does the AWS Meter Data Analytics Quick Start replace the existing MDMS from ISVs that Utilities have today?
+
+*A.* AWS is an exceptionally customer obsessed company. 90% of what AWS builds is what AWS customers directly ask for, while the remaining 10% is focused on strategic interpretations of customer needs. Talking to our utility customers, they have expressed a need to breakdown the data silos and bring greater value out of their data; in this case meter data. Customers asked for AWS best practices for storing meter data in a data lake, and then help to quickly get started running analytcis and ML against that data to derive value for their own customers. This Quick Start (QS) directly answers these needs. Utilities can hydrate the AWS data lake in this QS with Data from their existing systems and continue to evolve this Open Source QS to meet their business needs. We expect our customers, solution providers and system integrators to take advantage of this QS ,which allows utilities to start exploiting ML against their meter data in a matter of days.
```
Comment (sshvans) [inline docs/partner_editable/faq_troubleshooting.adoc]:
```suggestion
*A.* Utility customers have expressed a need to breakdown the data silos and bring greater value out of their data; in this case meter data. Customers asked for AWS best practices for storing meter data in a data lake, and then help to quickly get started running analytics and ML against that data to derive value for their own customers. This Quick Start (QS) directly answers these needs. Utilities can hydrate the AWS data lake in this QS with Data from their existing systems and continue to evolve this QS to meet their business needs. We expect our customers, solution providers and system integrators to take advantage of this QS, which allows utilities to start exploiting ML against their meter data in a matter of days.
```

File: docs/partner_editable/faq_troubleshooting.adoc [hunk 2]
```diff
@@ -11,10 +11,117 @@ WARNING: When you set *Rollback on failure* to *Disabled*, you continue to incur
...
+
+*A.* A reference of the internal data structure can be found in the https://aws-quickstart.github.io/quickstart-aws-utility-meter-data-analytics-platform/#_table_schemas[Documentation]. The internal data schema has been the result of working backworks from the requirements and talking to partners with a long history of industry experience. It should be well suited to implement future use cases. The reason for separating the Data Lake into three layers can be found in the https://aws.amazon.com/blogs/industries/aws-releases-smart-meter-data-analytics-platform/[blog article].
+
+---
+
+*Q.* Are there any other solutions like this that you are aware of, either from other cloud providers, or on-premises solutions?
+
+*A.* Some of our ISV and SI partners are attempting this. We are working with them to adopt our QS as a starting point. When it comes to On-prem, every utility has meter data and an MDM, and do some degree of analytics against the data. Very few do any form of ML against the data. Most want to do more, and do it in the cloud, to free themselves from the costs and constraint of on-prem analytics and ML.
```
Comment (sshvans) [inline docs/partner_editable/faq_troubleshooting.adoc]:
```suggestion
*A.* Some of the AWS partners are working to adopt the Quick Start as a starting point and building on top of it. When it comes to on-premises, most utility has meter data and an MDM, and do some degree of analytics against the data. Very few do any form of ML against the data. Most want to do more, and do it in the cloud, to free themselves from the costs and constraint of on-premises analytics and ML.
```

File: docs/partner_editable/faq_troubleshooting.adoc [hunk 3]
```diff
@@ -11,10 +11,117 @@ WARNING: When you set *Rollback on failure* to *Disabled*, you continue to incur
...
+
+*Q.* Have any financial information been included for any of the forecasts, e.g., to forecast the most cost effective generation source to cover peak loads and prevent brown-outs/black-outs?
+
+*A.* No, the only forecast use case supported right now is meter usage.
+
+---
+
+*Q.* Since meter data is time series data, why is the Quick Start not using TimeStream as a data store?
 
+*A.* Timestream was not available at the event of the MDA. We have plans to look into Timestream in the future. Timestream is a good option for certain use cases, i.e. real time dashboards.
```
Comment (sshvans) [inline docs/partner_editable/faq_troubleshooting.adoc]:
```suggestion
*A.* Timestream was not available at the time of the publication of this Quick Start. We are looking into updating it in future iterations. Timestream is a good option for certain use cases, i.e. real time dashboards.
```

**Review by sshvans — CHANGES_REQUESTED** [review]
Please see my comments.

File: docs/partner_editable/faq_troubleshooting.adoc [hunk 4]
```diff
@@ -11,12 +11,118 @@ WARNING: When you set *Rollback on failure* to *Disabled*, you continue to incur
 
 For additional information, see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/troubleshooting.html[Troubleshooting AWS CloudFormation^] on the AWS website.
 
+---
+
 *Q.* I encountered a size limitation error when I deployed the AWS CloudFormation templates.
 
-*A.* We recommend that you launch the Quick Start templates from the links in this guide or from another S3 bucket. If you deploy the templates from a local copy on your computer or from a location other than an S3 bucket, you might encounter template size limitations. For more information about AWS CloudFormation quotas, see the http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cloudformation-limits.html[AWS documentation^].
+*A.* We recommend that you launch the Quick Start templates from the links in this guide or from another S3 bucket. If you deploy the templates from a local copy on your computer or from a location other than an S3 bucket, you might encounter template size limitations. For more information about AWS CloudFormation quotas, see the http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cloudformation-limits.html[AWS documentation].
```
Comment (MarciaRieferJohnston) [inline docs/partner_editable/faq_troubleshooting.adoc:18]: Quick note—Here's the latest version of this answer as it has been edited in our current boilerplate: 

Launch the Quick Start templates from the links in this guide or from another S3 bucket. If you deploy the templates from a local copy on your computer or from a location other than an S3 bucket, you might encounter template-size limitations. For more information, see http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cloudformation-limits.html[AWS CloudFormation quotas^].

**Review by annaone — APPROVED** [review]

**Pull request merged**